backend/src/test/java/de/neuefische/cgnjava222/zoomanagement/zoo/plant/PlantIntegrationTest.java

```diff
@@ -30,4 +33,49 @@ void isGetPlantsListSize2() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$", hasSize(0)));
     }
-}
\ No newline at end of file
+
+    @Test
+    @DisplayName("addOnePlant")
+    @DirtiesContext
+    void addOnePlantAndCheckTheReturnLengthIfIt1() throws Exception {
```

Comment:
maybe just `testAddOnePlant` 

```suggestion
    void addOnePlantAndCheckTheReturnLengthIfIt1() throws Exception {
```